packages/cozy-search/src/components/AssistantMobile.jsx

```diff
@@ -28,7 +29,7 @@ export const AssistantMobile = ({ componentsProps }) => {
         />
       }
       type="button"
-      onClick={() => navigate('connected/search')} // FIXME this route is related to home app
+      onClick={() => navigate(`connected/search?returnPath=${pathname}`)} // FIXME this route is related to home app
```

Comment:
comment is still relevant?

If think (not sure) we can use param in navigate to use queryparam and avoid using `?`

Reply:
Comment is still relevant I hesitated to try to fix it but want to do other things today

Reply:
and what about using query params?